en/user_guide/media.rst

```diff
@@ -0,0 +1,60 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel.
```

Comment:
This introduction is very light, you don't describe the functionnality, which should be the first step in a user guide.

Maybe you could present a lot deeper what is the purpose of the media library: you talk about media, you should inisit on the fact that it's concerning only contributed media, not media used in the FO templating.

You could also talk here about formats supported natively by Open Orchestra.

Lastly, you could talk about the dynamic resize done when uploading a media, to have the best rendering in Front Office.